src/stavenote.ts

```diff
@@ -978,33 +1062,33 @@ export class StaveNote extends StemmableNote {
       non_displaced_x: nonDisplacedX,
       highest_line: highestLine,
       lowest_line: lowestLine,
-      highest_displaced_line: highestDisplacedLine,
-      lowest_displaced_line: lowestDisplacedLine,
+      highest_displaced_line: highestDisplacedLine ?? 0,
```

Comment:
I think these are wrong -- if there's no displaced lines, then returning `0` would be incorrect -- better to return `undefined`.